### ISSUE TITLE
EarthTime

### ISSUE THREAD

**Issue by taramk** [darwinkitdev/upwork-demos#12]

**Comment by ericmaciel** [comment]
Estimated time: 10h

- I'm thinking of using the CLGeocoder from the CoreLocation framework to search by city.
- There is the possibility of adding more than one item in the menu bar instead of opening multiple instances of the app and each one can have its own menu. This would require options to add and remove items from the menu bar. However, it would be easier to store and restore the information.

**Comment by taramk** [comment]
@ericmaciel re:multiple instances in one app:
- do you need mockups for this? I can't picture exactly how that might work
- is that included in the estimate and if not how much extra time would it be?

**Comment by ericmaciel** [comment]
Yes, it's included in the estimate. And I can implement it without the mock.

It would work in much the same way. What would change is that instead of opening a new instance of the app to add another time zone to the menu bar, you would click on an option in the menu to add a new time zone. This would create a new menu in the menu bar for that new time zone.

I couldn't think of a simple way to store the time zone of each instance so that it would be possible to restore the state of all of them later. It's not impossible, but it can get complex. That's why I thought of this other possibility, of having multiple time zones in the same instance.


Just as a side note, I can think of other ways of interacting with a menu in the menu bar (though Apple doesn't endorse these because they're "hidden"):
- Each mouse button can show a different menu. The main button could show the main menu (as in the mock) and the secondary button could show a menu with options to add a new time zone or remove the selected one.
- Dragging and dropping the menu button out of the menu bar, while pressing command, could be used to remove a time zone.
